index.html

```diff
@@ -11,5 +11,19 @@
 		<h4>Multiple Pages.</h4>
 		<h4>Motion.</h4>
 		<h4>Slide Show of kids.</h4>
+	<div>
+		<a href= "http://www.fieldmuseum.org/?gclid=CKr98OushcYCFRCqaQodnGUA2Q"> Take a trip into <span>History!</span></a>	
+	</div>
+	<table>
+		<thead><th colspan="2">Family Photo's!</th></thead>
+		<tr>
+		<td><img src="https://scontent-ord1-1.xx.fbcdn.net/hphotos-xft1/v/t1.0-9/11133785_10205739067267913_5019331051956596041_n.jpg?oh=426a7e6c629b0cb736819087dd1f309e&oe=55F94258"/></td>
+		<td><img src="https://scontent-ord1-1.xx.fbcdn.net/hphotos-xtf1/v/t1.0-9/10629593_10203880358721361_9071887968353752496_n.jpg?oh=172f5f543bc34b1dc09dcc76e5512117&oe=56036CC8"/></td>
+		</tr>
+		<tr>
+			<td><img src="https://scontent-ord1-1.xx.fbcdn.net/hphotos-xtp1/v/t1.0-9/10649808_10203880357561332_3445767377069126474_n.jpg?oh=5578093883e9eb72743e55b70c4303e0&oe=562CC8CB"/></td>
+			<td><img src="https://scontent-ord1-1.xx.fbcdn.net/hphotos-frc3/v/t1.0-9/526892_4481139478577_651037037_n.jpg?oh=62834ed93d3ba608c0c7b772cca91cac&oe=55E95620"/></td>
+		</tr>
+	</table>
```

Comment:
All the `&`'s in the `img src`'s should be HTML encoded … not sure if you've covered anything like that in you tutorials, but instead of just `&` use `&amp;` … same thing goes for apostrophes (like in `Family Photo's` … though really it's not possessive and should probably just be `Family Photos`). If you want to encode an apostrophy it's `&#39;` or `&quot;` ([reference](http://stackoverflow.com/a/2083770)).